64_allowlist_signing.md

```diff
@@ -0,0 +1,290 @@
+- Then, it will verify the inclusion proof using the results returned from the transparency log. If valid, allowlist loading continues; if not, it fails with an error.
+
+### Test Plan
+
+<!--
+**Note:** *Not required until targeted at a release.*
+
+Consider the following in developing a test plan for this enhancement:
+- Will there be e2e and integration tests, in addition to unit tests?
+- How will it be tested in isolation vs with other components?
+
+No need to outline all of the test cases, just the general strategy.  Anything
+that would count as tricky in the implementation and anything particularly
+challenging to test should be called out.
+
+All code is expected to have adequate tests (eventually with coverage
+expectations).
+-->
+
+- A new test should be written to ensure that the allowlist loading process checks for inclusion proof in a transparency log.
+
+
+### Upgrade / Downgrade Strategy
+
+<!--
+If applicable, how will the component be upgraded and downgraded? Make sure
+this is in the test plan.
+
+Consider the following in developing an upgrade/downgrade strategy for this enhancement
+-->
+
+Since this feature will take the form of a new option in Keylime’s tenant command, there should be no adverse impacts to upgrading or downgrading.
+
+### Dependency requirements
+
+<!--
+If your new change requires new dependencies, please outline and demonstrate that your selected dependency
+is well maintained and packaged in Keylime's supported Operating Systems (currently Debian Stable
+and as of time writing Fedora 32/33).
+
+During code implementation you will also be expected to add the package to CI , the keylime ansible role and
+keylimes main installer (`keylime/installers.sh`).
+
+If the package is not available in the supported Operated systems, the PR will not be merged into master.
+
+Adding the package in `requirements.txt` is not sufficent for master which is where we tag releases from.
+
+You may however be able to work within an experimental branch until a package is made available. If this is
+the case, please outline it in this enhancement.
+
+-->
+
+The cryptography and web interaction required involve the use of several Python libraries, most notably the `cryptography.*` library, though these should be available through Python’s package system if not installed already.
+
+## Drawbacks
+
+<!--
+Why should this enhancement _not_ be implemented?
+-->
+
+No drawbacks are known of.
+
+## Alternatives
+
+<!--
+What other approaches did you consider and why did you rule them out?  These do
+not need to be as detailed as the proposal, but should include enough
+information to express the idea and why it was not acceptable.
+-->
+
+There are more or less no alternatives (at present) for Sigstore as a transparency log.
```

Comment:
The "more or less" seems wishy washy. Maybe something like "Sigstore is the best known suitable transparency log for this kind of application. While the public-good version is a great option, users can always run their own internal versions for enhanced control and security. And our design does not preclude the use of other compatible transparency logs in the future". Or something like that.